daemon/mgr/cri_test.go

```diff
@@ -25,6 +25,22 @@ func TestParseUint32(t *testing.T) {
 	}
 }
 
+func TestToCriTimestamp(t *testing.T) {
```

Comment:
Please add some unit test cases to those returning a non-nil cases. 
Maybe in a follow-up.